src/SpectatorView.Native/SpectatorView.Compositor/Compositor/VideoEncoder.cpp

```diff
@@ -288,8 +288,8 @@ void VideoEncoder::WriteVideo(byte* buffer, LONGLONG timestamp, LONGLONG duratio
 
     // Copy frame to a temporary buffer and process on a background thread.
 #if HARDWARE_ENCODE_VIDEO
-    BYTE* tmpVideoBuffer = new BYTE[(int)(1.5f * frameHeight * frameWidth)];
-    memcpy(tmpVideoBuffer, buffer, (int)(1.5f * frameHeight * frameWidth));
+    BYTE* tmpVideoBuffer = new BYTE[(int)(FRAME_BPP_NV12 * frameHeight * frameWidth)];
```

Comment:
>FRAME_BPP_NV12 [](start = 42, length = 14)

NV12 is 4:1:1 if i remember correctly. Is this change mainly for cleanliness?

Reply:
Yes, this was part of cleaning up the code that used NV12 pixel format data, since I was initially confused by the 1.5 value and why it was being used